### DISCUSSION TITLE
Facing RabbitMQ ui render issue while upgrading rabbitMQ engine from 3.10.20 to 3.12.13

### DISCUSSION THREAD

**Discussion by Ankita-Maurya** [rabbitmq/rabbitmq-server#11812] [Questions] [answered]
I am trying to upgrade RabbitMQ engine version from 3.10.20 to 3.12.13. There are no UI changes at our end. Also the below error seems to be from ejs library itself. Also rabbit engine 3.10.20 uses erlang version 25.3 and rabbit engine version 3.12.13 uses erlang version 25.3.2.9. Due to the below issues i cannot see queues, connections etc in rabbitMQ console. The old rabbit engine version 3.10.20 does not have any issues loading the UI

ReferenceError: ac is not defined ReferenceError: ac is not defined at Array.process (eval at compile (https://someDomain/js/ejs-1.0.min.js:1:6654), :229:2) at EJS.render (https://someDomain/js/ejs-1.0.min.js:1:1885) at format (https://someDomain/js/main.js:1149:21) at https://someDomain/js/main.js:455:24 at with_reqs (https://someDomain/js/main.js:1131:9) at https://someDomain/js/main.js:1127:17 at req.onreadystatechange (https://someDomain/js/main.js:1208:17)

**Comment by lukebakken** [comment, marked as answer]
Force-refresh your browser, or clear all cache and cookies.

**Comment by michaelklishin** [comment]
From the [Upgrade guide](https://www.rabbitmq.com/docs/upgrade#management-ui):

> After upgrading a cluster, it is highly recommended to clear browser cache, local storage, session 
> storage and cookies for the domain(s) used to access the management UI.
